docs/examples/DataSet/The-Experiment-Container.ipynb

```diff
@@ -338,13 +266,6 @@
    "source": [
     "In a similar way, we may of course add runs to the other two experiments. Now, the alert reader will have noticed that `exp_id` is a keyword argument in `new_data_set`. What is the default then? --The default is that the run is added to the experiment with the **highest** `exp_id` in the database."
```

Comment:
could you still please add a note that new_data_set here is used for the sake of excersise and should not be used in "real life"? :)

Reply:
Done